packages/widget/src/pages/SwapExecutionPage/SwapExecutionButton.tsx

```diff
@@ -48,11 +48,19 @@ export const SwapExecutionButton: React.FC<SwapExecutionButtonProps> = ({
   const isGoFast = useIsGoFast(route);
 
   const getDestinationAddreessUnsetText = useCallback(() => {
-    if (lastOperation.signRequired && route?.txsRequired === 2) {
+    const destinationChainIdHasSignRequired =
+      lastOperation.signRequired && lastOperation.fromChainID === route?.destAssetChainID;
+
+    if (destinationChainIdHasSignRequired && route?.txsRequired === 2) {
```

Comment:
adding the check that `lastOperation.fromChainID === route?.destAssetChainID` should ensure that we only show this message if the `destinationChainId` has `signRequired`